bom/pom.xml

```
@@ -11,7 +11,7 @@
 
   <groupId>org.camunda.spin</groupId>
   <artifactId>camunda-spin-bom</artifactId>
-  <version>1.23.0-SNAPSHOT</version>
+  <version>1.23.1-SNAPSHOT</version>
```

:x: As it's the `1.22`, the version should be `1.22.1` for all files.
```suggestion
  <version>1.22.1-SNAPSHOT</version>
```

---

Nice catch!